codegen/generator/nodejs/templates/functions.go

```diff
@@ -41,9 +41,24 @@ var (
 		"ToLowerCase":         commonFunc.ToLowerCase,
 		"ToUpperCase":         commonFunc.ToUpperCase,
 		"ToSingleType":        toSingleType,
+		"ListOfEnum":          listOfEnum,
 	}
 )
 
+func listOfEnum() string {
+	schema := generator.GetSchema()
+
+	var result []string
+
+	for _, t := range schema.Types {
+		if t.Kind == introspection.TypeKindEnum && !strings.HasPrefix(t.Name, "__") {
+			result = append(result, t.Name)
+		}
+	}
+
+	return strings.Join(result, ", ")
```

Comment:
nit: IMO it'd be better for this to return a `[]string` and leave the formatting to the template

Reply:
It will make the template more verbose, I would prefer to keep it simple